client/components/FilterMenu/index.js

```diff
@@ -1,24 +1,24 @@
-import React, { useState } from 'react';
-import PropTypes from 'proptypes';
-import { connect } from 'react-redux';
-import { toggleMenu as reduxToggleMenu } from '@reducers/ui';
-import { makeStyles } from '@material-ui/core/styles';
+import React, { useState } from "react";
+import PropTypes from "proptypes";
+import { connect } from "react-redux";
+import { toggleMenu as reduxToggleMenu } from "@reducers/ui";
+import DateSelector from "@components/DateSelector";
+import Card from "@material-ui/core/Card";
+import CardHeader from "@material-ui/core/CardHeader";
+import CardContent from "@material-ui/core/CardContent";
+import IconButton from "@material-ui/core/IconButton";
+import ArrowDropDownIcon from "@material-ui/icons/ArrowDropDown";
+import ArrowDropUpIcon from "@material-ui/icons/ArrowDropUp";
+import Collapse from "@material-ui/core/Collapse";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+import GearButton from "../GearButton";
 
-import Card from '@material-ui/core/Card';
-import CardHeader from '@material-ui/core/CardHeader';
-import CardContent from '@material-ui/core/CardContent';
-import IconButton from '@material-ui/core/IconButton';
-import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
-import ArrowDropUpIcon from '@material-ui/icons/ArrowDropUp';
-import Collapse from '@material-ui/core/Collapse';
-import Typography from '@material-ui/core/Typography';
-import GearButton from '../GearButton';
-
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles((theme) => ({
   card: {
     width: 300,
     backgroundColor: theme.palette.primary.main,
-    position: 'absolute',
+    position: "absolute",
```

Comment:
changing strings in this file to use doublequote introduced a bunch of linting errors. I think it's passing the pull request checks because the `--fix` flag was added to the lint script. `npm run lint` before your next push should revert these changes.

Reply:
Yeah. I think we should have linitng as pre commit tasks for git.

Reply:
Sounds good to me, will get that pre commit hook in.

Reply:
I already have a ticket for it. https://github.com/hackforla/311-data/projects/10#card-53425622